src/js/consts.js

```diff
@@ -44,14 +45,12 @@ export const visibilityFunctions = {
     const userPermissions = await insights.chrome.getUserPermissions();
     return userPermissions && permissions.every((item) => userPermissions.find(({ permission }) => permission === item));
   },
-  apiRequest: async ({ url, method, ...options }) => {
-    // TODO: add caching
-    return instance.axios({
+  apiRequest: async ({ url, method, accessor, ...options }) =>
+    instance({
       url,
       method: method || 'GET',
       ...options,
-    });
-  },
+    }).then((response) => (accessor ? get(response?.data || {}, accessor) : response)),
```

Comment:
Since we are using `interceptors` you should return `response.data`
```suggestion
    }).then((response) => (accessor ? get(response?.data || {}, accessor) : (response?.data || response)),
```